src/HeadTag.js

```diff
@@ -25,7 +25,7 @@ export default class HeadTag extends Component {
     this.setState({ canUseDOM: true });
 
     const { tag, children, ...rest } = this.props; // eslint-disable-line react/prop-types
-    const ssrTags = document.head.querySelector(`${tag}${buildSelector(rest)}[data-reactroot=""]`);
+    const ssrTags = document.head.querySelector(`${tag}${buildSelector(rest)}[data-ssr=""]`);
```

Comment:
What do you think about naming this `data-rh` instead (react-head)?

Reply:
feel free (remember to duplicate change to the render method)